kubernetes.go

```diff
@@ -149,11 +149,13 @@ type KubernetesClusterConfig struct {
 
 // KubernetesClusterPoolConfig is used to create a new cluster pool
 type KubernetesClusterPoolConfig struct {
-	Region           string `json:"region,omitempty"`
-	ID               string `json:"id,omitempty"`
-	Count            int    `json:"count,omitempty"`
-	Size             string `json:"size,omitempty"`
-	PublicIPNodePool bool   `json:"public_ip_node_pool,omitempty"`
+	Region           string            `json:"region,omitempty"`
+	ID               string            `json:"id,omitempty"`
+	Count            int               `json:"count,omitempty"`
+	Size             string            `json:"size,omitempty"`
+	Labels           map[string]string `json:"labels,omitempty"`
```

Comment:
IIRC, the udpate endpoint doesn't support these fields(labels, taints, annotations) yet . Should we have a comment to explicitly say that till we add support?

Reply:
Also, the annotations field is missing?

Reply:
The MR is done, This need to be wait for the merge